app/views/admin/events/_proposal.html.haml

```diff
@@ -17,7 +17,7 @@
           %b Type
         %td
           .dropdown
-            = link_to '#', class: 'dropdown-toggle', id: 'type-dropdown' do
+            = link_to '#', class: 'dropdown-toggle', id: 'type-dropdown','data-toggle'=>'dropdown' do
```

Comment:
It would be awesome if you can also add spaces:
* after the comma
* before =>
* after =>